pdns/dnsdist.cc

```diff
@@ -273,11 +273,12 @@ bool fixUpResponse(char** response, uint16_t* responseLen, size_t* responseSize,
   }
 
   if (ednsAdded || ecsAdded) {
-    char * optStart = NULL;
+    uint16_t optStart = NULL;
```

Comment:
Initializing this to `NULL` is confusing

Reply:
Indeed!